man/systemd-run.xml

```diff
@@ -325,6 +325,11 @@
         to the terminal <command>systemd-run</command> is invoked on, via a pseudo TTY device. This allows running
         programs that expect interactive user input/output as services, such as interactive command shells.</para>
 
+        <para>This option will result in <command>systemd-run</command> synchronously waiting for
+        the transient service to terminate, similar to specifying <option>--wait</option>. If specified
+        along with <option>--wait</option>, <command>systemd-run</command> won't exit when manually disconnecting
+        from the pseudo TTY device.</para>
```

Comment:
The tenses are messed up here. But the surrounding paragraphs are also not great, so let's merge this, and work on the grammar later.